document.js

```diff
@@ -0,0 +1,30 @@
+function document(body, map){
+return `
+    <html lang="en">
+    <head>
+        <meta charSet="UTF-8"/>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+        
+       
+<script src='https://api.mapbox.com/mapbox-gl-js/v2.2.0/mapbox-gl.js'></script>
+<link href='https://api.mapbox.com/mapbox-gl-js/v2.1.1/mapbox-gl.css' rel='stylesheet' />
+<link href='/css/style.css' rel='stylesheet' ></link>
+
+        <title>Document</title>
+
+    </head>
+
+    <div className='map'></div>
```

Comment:
It's only `className` in JSX or in the browser - everywhere else this attribute is `class` =]

I think this should be an ID anyway, the `container` option for your map initialisation specifies `container: 'map'` so that will probably do a lookup by ID rather than class name.